src/Tasks/Microsoft.Common.CurrentVersion.targets

```diff
@@ -324,6 +324,8 @@ Copyright (C) Microsoft Corporation. All rights reserved.
 
   <PropertyGroup>
     <GenerateBindingRedirectsOutputType Condition="'$(OutputType)'=='exe' or '$(OutputType)'=='winexe'">true</GenerateBindingRedirectsOutputType>
+    <!-- It would be a breaking change to automatically turn on binding redirects for existing projects, so turn them on only when opting into a new framework. -->
+    <AutoGenerateBindingRedirects Condition="'$(AutoGenerateBindingRedirects)' == '' and '$(TargetFrameworkIdentifier)' == '.NETFramework' and '$(TargetFrameworkVersion.TrimStart(vV))' >= '4.7.2'">true</AutoGenerateBindingRedirects>
```

Comment:
What about class libraries? [We talked about not wanting to turn this feature on](https://github.com/dotnet/designs/blob/master/accepted/automatic-redirects-by-default/automatic-redirects-by-default.md#class-libraries) for class libraries unless they are unit test projects.